### ISSUE TITLE
Labels from old version are not shown

### ISSUE THREAD

**Issue by mahype** [awsmug/shipcloud-for-woocommerce#104]
A user who have updated to 1.3.1 has lost all his old labels and can't see anymore if there was package send or not. Can we test and reproduce this?

- see difference here: https://github.com/awsmug/shipcloud-for-woocommerce/compare/1.3.0...1.3.1

**Comment by andrecedik** [comment]
Do we know if the data is still in the DB? You could query the api to [get all shipments for a user](https://developers.shipcloud.io/reference/#getting-a-list-of-shipments). The you could use the returned `reference_number` to map those shipments to each order to do a kind of _re-import_.

**Comment by ScreamingDev** [comment]
I added a diff view for this version @mahype in the issue description
but this has nothing common with the problem.

TL;DR: Can't reproduce.

What I did:

- `git checkout 1.2.0` && vanilla WordPress
- Setup WooCommerce && shipcloud
- Generate order
- Generate label mark it as delivered via database (meta: `shipment_abc_status = shipment.tracking.delivered`)
- Generate label and leave it there (just to have another one).

Then I checked:

- `git checkout 1.2.1` & all labels there & still delivered
- 1.3.0 & labels there still delivered
- 1.4.0 & labels there still delivered
